### PR TITLE
run in batch -- minor tweak

### PR DESCRIPTION
**Pull request by vegetable68**
- deleted not necessary files
- fixed bug to read in batchno argument

### CODE REVIEW

File: conversation_reconstruction_dataflow/ingest_revisions/dataflow_main.py [hunk 1]
```diff
@@ -131,5 +110,27 @@ def process(self, element):
 
 if __name__ == '__main__':
   logging.getLogger().setLevel(logging.INFO)
-  run()
+  parser = argparse.ArgumentParser()
```
Comment (nthain) [inline conversation_reconstruction_dataflow/ingest_revisions/dataflow_main.py]: Why did you move the argument parsing to main?

Comment (vegetable68) [reply conversation_reconstruction_dataflow/ingest_revisions/dataflow_main.py]: I was trying to find the reason why it was not parsing one of the arguments, this is not the reason in the end. Should I move it back?
